managers/models.py

```diff
@@ -25,7 +26,7 @@ def filter_completed(self) -> QuerySet:
 
 class Project(models.Model):
     name = models.CharField(max_length=MAX_NAME_LENGTH)
-    start_date = models.DateField()
+    start_date = models.DateField(help_text=MESSAGE_FOR_CORRECT_DATE_FORMAT)
```

Comment:
Doesn't it require migration?

Reply:
Sure it does. Please add migration and add in front of this commit name `[M]`

Reply:
Ok, so I will wait for final decision and then I will add migrations, too.